logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/type/UserTypeMapper.kt

```diff
@@ -79,4 +92,13 @@ interface UserTypeMapper<T> {
 
     private fun selfUserIsTeamMember(selfUserTeamId: String?) = selfUserTeamId != null
 
+    fun teamRoleCodeToUserType(permissionCode: Int?): T = when(permissionCode) {
```

Comment:
I think this can be private 🤔  ?

Reply:
No, because it is used in mapping team members